### ISSUE TITLE
Customize colors possible?

### ISSUE THREAD

**Issue by bgarrant** [jsmecham/atom-terminal-tab#34]
Is there any way to do a simple background color change?  I am using One Dark and just wanted to match Terminal Tab to the theme if possible.

![screen shot 2018-01-08 at 3 07 31 pm](https://user-images.githubusercontent.com/2436728/34690067-ad35d9ee-f485-11e7-9a5b-1229ddef2292.png)

**Comment by acondiff** [comment]
@jsmecham, I submitted a PR for this a while ago. did it ever get merged in?

**Comment by bgarrant** [comment]
@acondiff Does not seem to be.

**Comment by bgarrant** [comment]
@jsmecham @acondiff This would be great as it is the only Terminal plugin that use Docks.

**Comment by bgarrant** [comment]
@jsmecham  @acondiff is there a way to just match the currently selected theme? I am using One Dark.

**Comment by bgarrant** [comment]
@acondiff What are the configuration options?  Do I add to Atom's styles/less?

**Comment by bgarrant** [comment]
I just tried this but nothing changed
```
.terminal-view .xterm {
  background-color: #223239;
  color: #fff;

  ::selection {
    background-color: #ececec;
  }

  .terminal-cursor {
    background-color: #fff;
  }
}
```

**Comment by bgarrant** [comment]
@jsmecham I tried this also with no luck. What do I need to change?

```
.terminal-view .xterm {
    background-color: #282C34;
    color: #ABB2BF;

    ::selection {
        background-color: #9196A1;
    }

    .terminal-cursor {
        background-color: #528BFF;
    }
}
```

**Comment by bgarrant** [comment]
This works but is there a cleaner way?

```
.terminal {
    background-color: #282C34;
    color: #fff;
}
.terminal .xterm-viewport {
    background-color: #282C34;
}
```

**Comment by jsmecham** [comment]
As @acondiff mentions, he did submit the PR. I have merged it and it is active on master, but I have not yet cut a release. There are some other bugs I am hoping to fix before then, but I may cut a release in the interim. Stay tuned.

**Comment by bgarrant** [comment]
Thanks @jsmecham . Love the plugin.  If I get the Master do you have a list of the variables I can use? I was trying to find that.

**Comment by jsmecham** [comment]
@bgarrant You should be able to override the classes defined in https://github.com/jsmecham/atom-terminal-tab/blob/master/styles/terminal.less if the theme matching isn't exactly what you're after.

**Comment by jsmecham** [comment]
I've just released [v0.3.0](https://github.com/jsmecham/atom-terminal-tab/tree/v0.3.0) which includes theme matching.
